devolutions-gateway/src/service.rs

```diff
@@ -97,7 +97,7 @@ impl GatewayService {
             match result {
                 Ok(_) => info!("XMF native library loaded and installed"),
                 Err(error) => warn!(
-                    %error,
+                    ?error,
```

Comment:
question: Why did you change that? IIRC there is no reason to use the Debug representation at this place.